navScript.js

```diff
@@ -13,4 +13,56 @@ function myFunction() {
         }
       }
     }
-  }
\ No newline at end of file
+  }
+
+  // function validarEmail(){
+
+  //   var txtmail = document.getElementsByName("email");
+  //   var regex = /^([a-z\d\.-]+)@([a-z\d-]+)\.([a-z]{2,3})$/ 
+
+  //   if (!regex.test(txtmail)) {
+  //       document.getElementById("mssg").innerHTML = "Format is not valid";
+  //   } else {
+  //     document.getElementById("mssg").innerHTML = "";
+  //   }
+  
+  // }
```

Comment:
- [ ] As requested by the previous reviewer, Kindly make sure to remove the commented code in your code base.

Reply:
I'm so sorry, I forgot to commit the change, I'm doing it.

thanks a lot!